assets/src/edit-story/components/form/color/color.js

```diff
@@ -57,23 +70,30 @@ function ColorInput({
 
   return (
     <Container aria-label={containerLabel}>
-      <ColorPreview
-        onChange={onChange}
-        hasGradient={hasGradient}
-        hasOpacity={hasOpacity}
-        value={value}
-        label={label}
-        colorPickerActions={colorPickerActions}
-        changedStyle={changedStyle}
-      />
+      <InputWrapper>
+        <ColorInput
+          onChange={onChange}
+          hasGradient={hasGradient}
+          hasOpacity={hasOpacity}
+          value={value}
+          label={label}
+          colorPickerActions={colorPickerActions}
+          changedStyle={changedStyle}
+        />
+      </InputWrapper>
       {hasOpacity && (
-        <OpacityPreview value={value} onChange={handleOpacityChange} />
+        <>
+          <Space />
+          <InputWrapper>
+            <OpacityInput value={value} onChange={handleOpacityChange} />
+          </InputWrapper>
+        </>
```

Comment:
I don't think this should show for a mixed color - like this one (mix of gradient and solid, one with 100% opacity, the other not):

![Screen Shot 2021-02-24 at 12 51 46](https://user-images.githubusercontent.com/637548/109043577-7d5c2300-769f-11eb-9e5f-f7114844688b.png)